WordPress/PHPCSHelper.php

```diff
@@ -112,4 +112,31 @@ public static function get_tab_width( File $phpcsFile ) {
 		return $tab_width;
 	}
 
+	/**
+	 * Check whether the `--ignore-annotations` option has been used.
+	 *
+	 * @since 0.13.0
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcsFile Optional. The current file being processed.
+	 *
+	 * @return bool True if annotations should be ignored, false otherwise.
+	 */
+	public static function ignore_annotations( File $phpcsFile = null ) {
+		// @TODO THIS NEEDS SOME CAREFUL & THOROUGH TESTING!
```

Comment:
Oh darn! No, that comment should be removed. Was a WIP comment to myself & I have tested it since and confirmed it works as expected.